dm/dm-webui-guide.md

```diff
@@ -0,0 +1,126 @@
+
+The interface is as follows:
+
+![webui](/media/dm/dm-webui-preview-en.png)
+
+## Access method
+
+You can access DM WebUI from any master node of the DM cluster. The access port is `8261` by default and is the same as that of DM OpenAPI. An example pf an access address: `http://{master_ip}:{master_port}/dashboard/`.
+
+## Prerequisites
+
+To ensure that DM WebUI can display information properly, before using DM WebUI, make sure that the following operations or configuration have been completed:
+
++ Enable the DM OpenAPI configuration:
+
+    - If your DM cluster has been deployed using binary, enable the `openapi` configuration item in the configuration of the master node:
+
+        ```
+        openapi = true
+        ```
+
+    - If your DM cluster has been deployed using TiUP, add the following configuration to the topology file:
+
+        ```yaml
+        server_configs:
+          master:
+            openapi: true
+        ```
+
++ When deploying Grafana for the first time, make sure that the `monitoring_servers` and `grafana_servers` components have been correctly installed. You can expected to configure `grafana_servers` as follows:
+
+    ```
+    grafana_servers:
+      - host: 10.0.1.14
+        # port: 3000
+        # deploy_dir: /tidb-deploy/grafana-3000
+        config:       # Make sure that the TiUP version for tiup dm -v is later than v1.9.0.
+          auth.anonymous.enabled: true
+          security.allow_embedding: true
+    ```
+
+    If the IP and port of `grafana_servers` are not the default ones, you need to fill in the correct IP and port in the **Dashboard** interface.
+
++ If your DM cluster is upgraded from an old version, you need to manually modify the Grafana configuration:
+
+    1. Edit the `/{deploy-dir}/grafana-{port}/conf/grafana.ini` file as follows:
+
+        ```ini
+        [auth.anonymous]
+        enabled = true
+
+        [security]
+        allow_embedding = true
+        ```
+
+    2. Run `tiup dm reload` to make the new configuration effective.
+
+## Dashboard
+
+To see the monitoring of migration tasks, visit the **Dashboard** page. **Dashboard** is an embedded Grafana Dashboard that contains `Standard` and `Professional` views, each of which displays monitoring information from a standard perspective, or from a more professional perspective.
+
+## Migration
+
+**Migration** includes **Source**, **Task**, and **Replication Detail** pages.
+
+## Source
+
+Before creating a migration task, you need to create the data source information of the upstream for the replication task. You can create the upstream task configuration in the **Source** page. When creating tasks, pay attention to the following items:
+
+- If there is a switch between primary and secondary node, enable GTID in the upstream MySQL and set GTID to `True` when creating the upstream configuration; otherwise, the migration task will be interrupted during the switch (except for AWS Aurora).
```

Comment:
```suggestion
- If there is a auto failover between primary and secondary instance, enable GTID in the upstream MySQL and set GTID to `True` when creating the upstream configuration; otherwise, the migration task will be interrupted during the failover (except for AWS Aurora).
```